cavalry_tests/tests/test_cavalry.py

```diff
@@ -24,13 +26,22 @@ def test_cavalry(settings, as_admin, enable, posting, admin_user):
             status_code=201,
             json={"ok": True},
         )
-        content = client.get("/").content
+        resp = client.get("/streaming/" if streaming else "/")
+        if isinstance(resp, StreamingHttpResponse):
```

Comment:
Probably not that significant,
but could assert the response type according to the `streaming` value?

Reply:
Belt-and-suspenders! Just in case someone optimizes the `/streaming/` endpoint to not stream...